src/ui/studio/save-creation/index.js

```diff
@@ -287,12 +289,29 @@ const ConnectionUnconfiguredWarning = () => {
 
 const UploadForm = ({ opencast, uploadState, recordings, handleUpload }) => {
   const { t } = useTranslation();
+  let series = opencast.getFinalSeriesTitles();
+  let titles = [];
+  for (let keys of series.keys()){
+    let tmp = {value:keys,label:series.get(keys)};
+    titles.push(tmp);
```

Comment:
Three of these `let`s should be `const`s. Also why the `tmp` variable and not put that object literal immediately into `push`? The line is only a few chars longer. You could even break the object literal into multiple lines.